test/account.go

```diff
@@ -312,3 +312,34 @@ func AssertIdentityEqual(t require.TestingT, expected, actual *account.Identity)
 	assert.Equal(t, expected.User.ImageURL, actual.User.ImageURL)
 	assert.Equal(t, expected.User.URL, actual.User.URL)
 }
+
+// AssertIdentityObfuscated verifies that the `actual` identity was obfuscated, ie,
+// all the personal data were replaced with UUID values, except a few fields
+func AssertIdentityObfuscated(t require.TestingT, expected, actual *account.Identity) {
+	require.NotNil(t, expected)
+	require.NotNil(t, actual)
+	require.NotEmpty(t, actual.User)
+	assert.Equal(t, expected.ID, actual.ID)
+	// verify that all those fields have a UUID for value (ie, data was obfuscated)
+	for _, field := range []string{
+		actual.Username,
+		*actual.ProfileURL,
+		actual.User.Bio,
+		actual.User.Company,
+		actual.User.Email,
+		actual.User.FullName,
+		actual.User.ImageURL,
+		actual.User.URL,
+		actual.User.Cluster,
+		actual.User.FeatureLevel,
+	} {
+		_, err := uuid.FromString(field)
```

Comment:
Nice way of checking the obfuscated data! :)

Reply:
thanks ;)